google/auth/external_account_authorized_user.py

```diff
@@ -117,6 +118,7 @@ def __init__(
         self._client_secret = client_secret
         self._revoke_url = revoke_url
         self._quota_project_id = quota_project_id
+        self._scopes = scopes
```

Comment:
The condition on line 107 is wrong - if refresh_token is provided, we also need client_id, client_secret and token_url.

Reply:
Done